20240923.md

```diff
@@ -0,0 +1,68 @@
+# Meeting Minutes
+
+**Cardano Atlantic Council Meeting**
+
+**Date:** Septemer 23, 2024 - 18:30 UTC
+
+## **Attendees:** 
+- Jenny Brito
+- Adam Dean
+- Beatrice Anihiri
+- Richard McCracken
+- Thiago Nunes
+- Mike Hornan
+
+## **Purpose:** 
+- To dicuss and possibly reach a decision on the current info action known as the Ikigai proposal.
+- Prerequisite to the meeting was the reading and evaluation of the proposal prior to the meeting.
+- Setting up a legal entity.
+
+## **Discussion Points:**
+
+1. **Process to help decision making**
+- How to prevent group think and confirmation bias in the decision making process.
+- Preliminary voting thoughts on the current active Ikigai proposal.
+- Discussion on order of operations, tentative votes and people's thoughts.
+- First round the room for individucals voting preferences and rationales.
+- Review of the proposal and the relevant sections of the Interim Constitution.
+
+2. **Round 2 of discussion and voting**
+- Secend round the room temperature check to see if we have consensus.
+- Further discussion of the pros and cons of a Yes, No or Abstain vote.
+- Further review of the proposal and relevant secions of the Interim Constitution.
+- Possible Abstain decsion.
+
+
+3. **Disseneting Rationale**
+- The dissent to a Yes vote as opposed to an abstain vote provides support for the content of the proposal. This was determined to be a double edge sword since the purpose of the ICC is determine the 
+- Dissent to a Yes or Abstain vote. When CAC as an ICC member votes in any way on a proposal, it does not represent a commitment from CAC that each person agrees or disagrees with the content of the proposal, nor to abide or pledge to it. It simply means that the proposal is constitional or not as written.
+- Dissent to No was not needed as most people leaned Yes or Abstain.
+
+
+4. **Extensive discussion and review of the decision**
+- On track with methods of casting individual votes.
+
+5. **Vote Decision**
+- Abstain from all team members.
+- Rationale: The proposal requires no action (info or otherwise) from the ICC. It does not set a budget, name a process, impose a requirement, request funds, or change a paramter. So in this case it's best to abstain on this proposal as per Article 4, Secion 1 "responsible for ensuring that on-chain governance actions prior to enactment on-chain, are constitutional."
```

Comment:
```suggestion
- Rationale: The proposal requires no action (info or otherwise) from the ICC. It does not set a budget, name a process, impose a requirement, request funds, or change a parameter. So in this case it's best to abstain on this proposal as per Article 4, Section 1 "responsible for ensuring that on-chain governance actions prior to enactment on-chain, are constitutional."
```